pinot-core/src/main/java/org/apache/pinot/core/operator/combine/CombineOperatorUtils.java

```diff
@@ -70,5 +71,6 @@ public static void setExecutionStatistics(IntermediateResultsBlock resultsBlock,
     resultsBlock.setNumEntriesScannedInFilter(numEntriesScannedInFilter);
     resultsBlock.setNumEntriesScannedPostFilter(numEntriesScannedPostFilter);
     resultsBlock.setNumTotalDocs(numTotalDocs);
+    resultsBlock.setThreadCpuTimeNs(threadCpuTimeNs);
```

Comment:
Sorry, should have discussed this in previous PR. Can we please rename the method to `setExecutionThreadCpuTimeNs` ?

Reply:
done